typing/jkind.ml

```diff
@@ -1504,19 +1539,26 @@ module Const = struct
       (* for each mode, lower the corresponding modal bound to be that mode *)
       let parsed_modifiers = Typemode.transl_modifier_annots modifiers in
       let mod_bounds =
-        Mod_bounds.Create.f
-          { f =
-              (fun (type a) ~(axis : a Axis.t) ->
-                let (module A) = Axis.get axis in
-                let parsed_modifier =
-                  Typemode.Transled_modifiers.get ~axis parsed_modifiers
-                in
-                let base_bound = Mod_bounds.get ~axis base.mod_bounds in
-                match parsed_modifier, base_bound with
-                | None, base_bound -> base_bound
-                | Some parsed_modifier, base_modifier ->
-                  A.meet base_modifier parsed_modifier.txt)
-          }
+        let value_for_axis (type a) ~(axis : a Axis.t) : a =
+          let (module A) = Axis.get axis in
```

Comment:
This seems analogous to the situation in normalize

Reply:
analogously, this doesn't really show up on a profile, so I'm hesitant to make it more verbose without good reason.